pandas/util/_validators.py

```diff
@@ -468,7 +468,7 @@ def validate_endpoints(closed: str | None) -> tuple[bool, bool]:
     return left_closed, right_closed
 
 
-def validate_inclusive(inclusive: str | None) -> tuple[bool, bool]:
+def validate_inclusive(inclusive: str) -> tuple[bool, bool]:
```

Comment:
Could use `IntervalClosedType` from `_typing.py` here

Reply:
yes could be but i'm not generally keen on using Literals in internal validation routines but maybe no big deal here as we probably only pass on values from the public api where the variables are already declared as Literals.